src/3_branch.js

```diff
@@ -301,8 +310,28 @@ Branch.prototype['init'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
 		done(err, data && utils.whiteListSessionData(data));
 	};
 
+	var allTheWaysToSayHidden =
+		[{ 'hidden': 'visibilitychange' },
+		{ 'mozHidden': 'mozvisibilitychange' },
+		{ 'msHidden': 'msvisibilitychange' },
+		{ 'webkitHidden': 'webkitvisibilitychange' }],
+
+	attachVisibilityEvent = function(data) {
+		for(var index = 0; index < allTheWaysToSayHidden.length; index++) {
+				var hidden = Object.keys(allTheWaysToSayHidden[index])[0],
```

Comment:
what about just making the allTheWaysToSayHidden an array, so like
```
// be sure ot add a comment here explaining what these are
var allTheWaysToSayHidden = [['hidden', 'visibilitychange'],['mozHidden', 'mozvisibilitychange'], ...;
```
that way you prevent the nasty Object.keys

alternately tbh since there are only four of them, it might be easiest to just spell out each one